### PR TITLE
For commenting on 00-00-Preface.ipynb (part of review-ee0cbc6)

### PR DESCRIPTION
**Pull request by mwcraig**

### CODE REVIEW

File: notebooks/00-00-Preface.ipynb [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The purpose of this text is to walk through image reduction and photometry using\n",
+    "Python, especially Astropy and its affiliated packages. It assumes some basic\n",
+    "familiarity with astronomical images and with Python. The inspiration for this\n",
+    "work is a pair of guides written for IRAF, [\"A User's Guide to CCD Reductions with IRAF\" (Massey 1997)](http://www.ifa.hawaii.edu/~meech/a399/handouts/ccduser3.pdf) and\n",
+    "[\"A User's Guide to Stellar CCD Photometry with IRAF\" (Massey and Davis 1992)](https://www.mn.uio.no/astro/english/services/it/help/visualization/iraf/daophot2.pdf).\n",
```
Comment (kelle) [inline notebooks/00-00-Preface.ipynb]: We really should have properly archived versions of these documents. maybe we can ask Phil to upload to Zenodo?

File: notebooks/00-00-Preface.ipynb [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    "\n",
+    "The focus is on optical/IR images, not spectra."
+   ]
+  },
+  {
```
Comment (kelle) [inline notebooks/00-00-Preface.ipynb]: I think there should be a subsection here titled something like Authors, Acknowledgements, Citation? That's a long title...but something like that here.

File: notebooks/00-00-Preface.ipynb (hunk 1 above)
Comment (mwcraig) [reply notebooks/00-00-Preface.ipynb]: Issue created at #141

File: notebooks/00-00-Preface.ipynb (hunk 2 above)
Comment (mwcraig) [reply notebooks/00-00-Preface.ipynb]: Have added several sections like this -- the revised version will be live soon. At the moment I've listed you under acknowledgments (as opposed to author or editor) but I don't think that captures your role accurately.

**Comment by mwcraig** [comment]
Closing this comment PR now that issue has been created to track last fix.

**Pull request closed without merging**